src/components/Profile/Profile/FeatureContainer/DoctorsFeature/ItemDoctorsFeature.tsx

```diff
@@ -29,7 +35,7 @@ const ItemDoctorsFeature: React.FC<IItemDoctorsProps> = ({
           <RemoveButton disabled />
         </Tooltip>
       ) : (
-        <RemoveButton onClick={() => deleteDate(idWorkTime)} />
+        <RemoveButton onClick={deleteDateTime} />
```

Comment:
```suggestion
        <RemoveButton onClick={() => deleteDate(idWorkTime || _id)} />
```

Reply:
btw, why are there 2 possible id's to delete?

Reply:
fixed, left only _id